ExtraDry/ExtraDry.Blazor/Components/Dry/DryTable.razor.cs

```diff
@@ -184,8 +184,25 @@ private void ToggleSelectAll()
         }
     }
 
+    private bool IsHierarchyList { 
+        get {
+            var listItem = InternalItems.FirstOrDefault(e => e.Item != null);
```

Comment:
This feels like a one-liner would be clearer...  E.g.
`=> (InternalItems.FirstOrDefault() as IHierarchyEntity) is not null`

Reply:
I had spread this out because I thought it was easier to read with the `if` statements. Happy either way, it's not the most complex of statements!